.gitignore

```diff
@@ -36,3 +36,7 @@ yarn-error.log*
 
 # localcache
```

Comment:
Do we need these comments?

Reply:
this concept has already been addressed, so this question should be addressed to @krzysztofwolski or @zaiste